en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#apiusage">APIUsage</a>]</td>
+<td>
+
+List containing the APIs usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageByBackendOverTime
+
+Represents API usage across APIs by each backend over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#apiusagebybackend">APIUsageByBackend</a>]</td>
+<td>
+
+List containing the API usage values across APIs by each backend. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageByGeoLocation
+
+Represents a single API usage across different countries.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usageByGeoLocation</strong></td>
+<td valign="top">[<a href="#usagebygeolocation">UsageByGeoLocation</a>]</td>
+<td>
+
+List containing usages of API in different countries. Returns an empty array if no data available.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageOverTime
+
+Represents a single API usage over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#apiusage">APIUsage</a>]</td>
+<td>
+
+List containing the API usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### AlertSummaries
+
+Represents alert summaries.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#alertsummary">AlertSummary</a>]</td>
+<td>
+
+List containing alert summary for each alert. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".